docs/admin/monitoring.rst

```diff
@@ -168,22 +168,78 @@ NodeStatus can be accessed using the JMX MBean object name
 NodeInfo MBean
 --------------
 
-The ``NodeInfo`` JMX MBean exposes information about the current node;
+The ``NodeInfo`` JMX MXBean exposes information about the current node;
 
-NodeInfo can be accessed using the JMX MBean object name
+NodeInfo can be accessed using the JMX MXBean object name
 ``io.crate.monitoring:type=NodeInfo`` and the following attributes:
 
- - ``ClusterStateVersion``
-
-   Provides the version of the current applied cluster state
-
- - ``NodeId``
-
-   Provides the unique identifier of the node in the cluster
-
- - ``NodeName``
-
-   Provides the human friendly name of the node
++-------------------------+---------------------------------------------------+
+| Name                    | Description                                       |
++=========================+===================================================+
+| ``NodeId``              | Provides the unique identifier of the node in the |
+|                         | cluster.                                          |
++-------------------------+---------------------------------------------------+
+| ``NodeName``            | Provides the human friendly name of the node.     |
++-------------------------+---------------------------------------------------+
+| ``ClusterStateVersion`` | Provides the version of the current applied       |
+|                         | cluster state.                                    |
++-------------------------+---------------------------------------------------+
+| ``ShardStats``          | Statistics about the number of shards located on  |
+|                         | the node.                                         |
++-------------------------+---------------------------------------------------+
+| ``ShardInfo``           | Detailed information about the shards located on  |
+|                         | the node.                                         |
++-------------------------+---------------------------------------------------+
+
+``ShardStats`` returns a `CompositeData`_ object containing statistics about
+the number of shards located on the node with the following attributes:
+
++-------------------+---------------------------------------------------------+
+| Name              | Description                                             |
++===================+=========================================================+
+| ``Total``         | The number of shards located on the node.               |
++-------------------+---------------------------------------------------------+
+| ``Primaries``     | The number of primary shards located on the node.       |
++-------------------+---------------------------------------------------------+
+| ``Replicas``      | The number of replica shards located on the node.       |
++-------------------+---------------------------------------------------------+
```

Comment:
I left 'RECOVERING' out on purpose as it would require to access the actual shard instance and cannot be retrieved from the cluster state. I can add it if desired.